config/optional/linkit.linkit_profile.default.yml

```diff
@@ -3,29 +3,30 @@ status: true
 dependencies:
   module:
     - node
-id: basic_html
-label: 'Basic HTML'
+id: default
+label: 'Default'
 description: ''
-attributes: {  }
 matchers:
   abb64004-44db-4254-8189-1a06a4b8c830:
     uuid: abb64004-44db-4254-8189-1a06a4b8c830
     id: 'entity:node'
     weight: 0
     settings:
-      result_description: Article
       bundles:
         article: article
       group_by_bundle: false
       include_unpublished: false
+      metadata: '[node:author] | [node:created:medium]'
+      substitution_type: canonical
+      limit: 100
   54dba6c0-a78b-4f93-8955-b083b930a2ff:
     uuid: 54dba6c0-a78b-4f93-8955-b083b930a2ff
     id: 'entity:media'
     weight: 0
     settings:
-      result_description: Media
       bundles:
-        gallery: gallery
         image: image
-        video: video
```

Comment:
Why is gallery and video missing, if I run update hooks from linkit I get 
```
uuid: 6d37c706-1e49-4fe6-9512-3acc22fa4858
langcode: en
status: true
dependencies:
  module:
    - node
_core:
  default_config_hash: JxIoOR0iB_8bRbUrta1Lr_CmfejEq9BXozXS8JA1JMA
id: basic_html
label: 'Basic HTML'
description: ''
matchers:
  abb64004-44db-4254-8189-1a06a4b8c830:
    uuid: abb64004-44db-4254-8189-1a06a4b8c830
    id: 'entity:node'
    weight: 0
    settings:
      bundles:
        article: article
      group_by_bundle: false
      include_unpublished: false
      substitution_type: canonical
      metadata: Article
  54dba6c0-a78b-4f93-8955-b083b930a2ff:
    uuid: 54dba6c0-a78b-4f93-8955-b083b930a2ff
    id: 'entity:media'
    weight: 0
    settings:
      bundles:
        gallery: gallery
        image: image
        video: video
      group_by_bundle: true
      substitution_type: canonical
      metadata: Media
```

Reply:
I removed gallery and video, because with d8.7 there is no canonical path for media anymore. So they should not be linked. For image it's different, because the actual image will be linked, not the canonical of the media item.

Reply:
Yes, I improved the metadata for new installs. The token possibility was not there in linkit4